bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -75,6 +75,10 @@ type KubeadmConfigSpec struct {
 	// +optional
 	Mounts []MountPoints `json:"mounts,omitempty"`
 
+	// BootCommands specifies extra commands to run very early in the boot process
+	// +optional
```

Comment:
```suggestion
	// BootCommands specifies extra commands to run very early in the boot process via the cloud-init bootcmd
	// module. This is typically run in the cloud-init.service systemd unit. This has no effect in Ignition.
	// +optional
```

Make this super-explicit, also indicates to Ignition users that it won't be used there.

Reply:
Thank you for the suggestion, I agree that it could've been clearer :+1: